admin/includes/languages/english/configuration.php

```diff
@@ -32,3 +32,71 @@
 define('TEXT_DATA_OUT_OF_RANGE', 'Data out of range');
 define('TEXT_MIN_GENERAL_ADMIN', 'The minimum value entered was not a whole number (integer).');
 define('TEXT_MAX_GENERAL_ADMIN', 'The value entered as a maximum was not a whole number (integer).');
+
+// These are the same now but may diverge in the future, so it's useful
+// to have separate constants 

```

Comment:
Can we collapse these 2 lines into one? makes for better readability instead of having a line-break in the middle of the thought